### ISSUE TITLE
Documentation: One of the sign post component is not getting appropriate focus with arrow key navigation due to which it's not clickable or actionable

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4656]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.1.1: Keyboard

**Test Environment:**
- Test URL: https://clarity.design/documentation/signposts
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. open the test URL.
2. Using arrow key navigation try to move focus on the signpost clickable icon mentioned below:

&lt;p&gt;Signposts should be used when you want to show a small amount of contextual help
    of information without taking the user out of the current context.
     Use sparingly as a supplemental element and not as a primary method of adding detail.
    &lt;clr-signpost&gt;
        &lt;clr-signpost-content *clrIfOpen=&quot;openState&quot;&gt;
            &lt;h3 style=&quot;margin-top: 0&quot;&gt;Inline signpost&lt;/h3&gt;
            &lt;p&gt;Position: &lt;code class=&quot;clr-code&quot;&gt;right-middle&lt;/code&gt;&lt;/p&gt;
        &lt;/clr-signpost-content&gt;
    &lt;/clr-signpost&gt;
&lt;/p&gt;

**Expected result:**
For the reference example provided in the documentation, clickable signpost icon should receive appropriate focus with arrow key navigation.

**Actual result:**
In the example provided, clickable icon is getting read along with the &lt;p&gt; content and Screen reader user is unable to move focus on the icon using arrow key navigation. Tab key navigation works fine.
